lib/engine/game/g_1854/step/merge_minors.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/base'
+
+module Engine
+  module Game
+    module G1854
+      module Step
+        class MergeMinors < Engine::Step::Base
+          def actions(entity)
+            return [] unless entity == current_entity
+            return [] unless entity.minor?
+            return [] unless @game.minor_mergers_allowed?
+
+            actions << ['merge']
```

Comment:
You haven't created the actions array yet, so can't append anything to it.
```suggestion
            actions = ['merge']
```